src/components/PeopleManagement/AddMembersModal/AddMembersModalContent.jsx

```diff
@@ -52,7 +52,7 @@ const AddMembersModalContent = ({
         <Col>
           <FormattedMessage
             id="people.management.add.members.modal"
-            defaultMessage="Only members registered with your organization can be added to a group."
+            defaultMessage="Only members registered with your organization can be added to a group. "
```

Comment:
Might recommend making the space between the text and the hyperlink a bit more explicit than a space within `<FormattedMessage>`. For example, perhaps adding `className="ml-1"` to the `Hyperlink` below, or alternatively wrapping the `FormattedMessage` and `Hyperlink` with a `Stack`, e.g.

```tsx
<Stack direction="horizontal" gap={1}>
  <FormattedMessage />
  <Hyperlink variant="inline">Learn more</Hyperlink>
</Stack>
```